libc/CMakeLists.txt

```diff
@@ -20,6 +21,13 @@ add_library(oelibasm OBJECT
     ${MUSLSRC}/math/x86_64/sqrtf.s
     ${MUSLSRC}/setjmp/x86_64/longjmp.s
     ${MUSLSRC}/setjmp/x86_64/setjmp.s)
+elseif(OE_TRUSTZONE)
+add_library(oelibasm OBJECT
```

Comment:
Indent, but otherwise LGTM since the list of files differs more than just by the arch subdirectory